packages/alfa-rules/test/sia-r62/rule.applicability.spec.tsx

```diff
@@ -352,7 +352,7 @@ test(`evaluate() is applicable to an <a> element with a <p> parent element that
       .withStyle(
         [
           "background",
-          "linear-gradient(to right, rgb(0% 0% 0%) 50%, rgb(0% 0% 93.33333%) 50%) 0% 0%",
+          "linear-gradient(to right, rgb(0% 0% 0%) 50%, rgb(0% 0% 93.33333%) 50%)",
```

Comment:
Fixing the `initial` value in `background-position-{x,y}` result in these being removed because we filter out initial values during serialisation.